src/components/EntityInfo.vue

```diff
@@ -243,12 +242,7 @@ export default class EntityInfo extends Mixins(AllowableActionsMixin, CommonMixi
 
   /** The business description. */
   get businessDescription (): string {
-    const corpTypeDescription = this.getCorpTypeDescription(this.getEntityType)
-
-    if (this.isSoleProp) {
-      return `${corpTypeDescription} / Doing Business As (DBA)`
```

Comment:
Are you certain about removing this? It was an explicit requirement from Yui. Please confirm.

Reply:
Looking at the design requirements in the ticket (12629), "DBA" should be removed once a SP has been registered/created.

This change also removed "DBA" when the business is still unregistered, which is incorrect.

![image](https://user-images.githubusercontent.com/10002889/177372997-62d57102-4022-43e7-807b-5b6d83378578.png)

Reply:
Is there a specific field you know off the top of your head which I can check to see if a business is registered or not?

Reply:
Yes. If there is a BUSINESS_ID in session storage then this is a business. Otherwise there should be a TEMP_REG_NUMBER in session storage, indicating that this is not a business yet. These are mutually exclusive.

https://github.com/bcgov/business-filings-ui/blob/493b73328f8377515832264f8740e0419ca551c8/src/App.vue#L221
https://github.com/bcgov/business-filings-ui/blob/493b73328f8377515832264f8740e0419ca551c8/src/App.vue#L231

Reply:
perfect, thanks!

Reply:
This will be fixed in https://github.com/bcgov/business-filings-ui/pull/357.